app/src/main/AndroidManifest.xml

```diff
@@ -46,13 +46,12 @@
         </activity>
         <activity
             android:name="com.github.pockethub.ui.gist.CreateGistActivity"
-            android:configChanges="orientation|keyboardHidden|screenSize">
+            android:configChanges="orientation|keyboardHidden|screenSize"
+            android:label="@string/create_gist">
```

Comment:
Otherwise, remove the setting of lable from the code and use <code>R.string.new_gist</code> here. Since that's what we used earlier.